.github/ISSUE_TEMPLATE/New_feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: New Feature Request
+about: Request for engineering work on a new feature
+title: 'New Feature Request'
+labels: 
+assignees: '@abbylindsay'
+
+---
+
+**Describe the work you would like completed:**
```

Comment:
Required fields should be marked as required here to prevent users from omitting items.